### DISCUSSION TITLE
Not able to run the first flutter command

### DISCUSSION THREAD

**Discussion by amitgu22** [ashishrawat2911/firekart_app#79] [General]
Could not find a file named "pubspec.yaml" in "C:\DreamProject\flutter_commerce".
pub finished with exit code 66

**Comment by warioddly** [comment]
> Could not find a file named "pubspec.yaml" in "C:\DreamProject\flutter_commerce".
> 
> pub finished with exit code 66

You can create this file (or take from another one) or otherwise create a project and assume that this project is started
